src/components/Footer.js

```diff
@@ -56,6 +66,7 @@ Footer.propTypes = {
   displayInformation: PropTypes.bool.isRequired,
   handleLocationSharedClick: PropTypes.func.isRequired,
   marker: PropTypes.bool.isRequired,
+  setVideoModal: PropTypes.func.isRequired
```

Comment:
This wouldn't be required, as the `setVideoModal` function is being created within the Footer.